analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/NullPointerDereference.CSharp8_NullableEnable.cs

```diff
@@ -0,0 +1,81 @@
+﻿#nullable enable
+
+using System;
+
+namespace Tests.Diagnostics.CSharp8
+{
+    public class NullForgivingOperator
+    {
+        object? field;
+        static object? staticField;
+        event EventHandler<object>? handler;
+
+        public void Local()
+        {
+            object? o = null;
+            o.ToString();                        // Noncompliant
+            o = null;
+            o!.ToString();                       // Compliant
```

Comment:
Let's add one line to prove we were able to learn from the `.ToString()` even when the `!` was in the way (should just work)
```suggestion
            o!.ToString();                       // Compliant
            o.ToString();                        // Compliant
```